### DISCUSSION TITLE
ms store app security question

### DISCUSSION THREAD

**Discussion by diffent** [MicrosoftEdge/WebView2Feedback#4017] [Q&A]
when you package up a UWP web app for the store, is there any kind of encryption of the package contents?
e.g. if we have html and javascript built into the Assets folder of the app, are they viewable as plain
text easily after the user downloads the app?

**Comment by novac42** [comment]
Thanks for reaching out. I've assigned this to a dev that can best answer your question.

**Comment by LiangTheDev** [comment]
There is no encryption of package contents. All html/script assets will be visible to the user as plain text in package install folder, which is C:\Program Files\WindowsApps\<package-full-name>.
